LICENSE.spdx

```diff
@@ -39,7 +39,7 @@ PackageFileName: ./plugins/wasi_nn/thirdparty/ggml
 PackageHomePage: https://github.com/ggerganov/llama.cpp
 PackageOriginator: Georgi Gerganov
 PackageLicenseDeclared: MIT
-PackageDownloadLocation: git://github.com/ggerganov/llama.cpp.git@b1383
+PackageDownloadLocation: git://github.com/ggerganov/llama.cpp.git
```

Comment:
We should also update `PackageFileName`.

Reply:
Just update the `PackageFileName` to the `CMakeLists.txt` file that downloaded `llama.cpp`.